### PR TITLE
how to create a simple Covid-19 dashboard.

### PR DESCRIPTION
**Pull request by AntoineSoetewey**
Thanks for contributing your submission to the e-Rum2020 CovidR contest.

**For CovidR contributors:**

- [x] The title of this PR reflects your contribution
- [x] The pull request only includes one new `.yml` file under `contest/contributions`
- [x] Make sure "Allow edits from maintainers" is checked
- [x] Send an email to covidrcontest@erum.io (for further direct communication down the road)
     - Subject: "Submission to CovidR: PR #00", where #00 is the number of this Pull Request
     - Content: A full link to this Pull Request
     - Optionally include your social media profiles (Twitter, LinkedIn, Facebook): we will use them to `@mention` you while sharing the contribution through our channels.

---

For CovidR maintainers:

- [x] Label the PR as "submission"
- [x] Check the validity of the submission
- [x] Assess completeness of metadata and possibly adjust the `.yml` file-name
- [x] Check the integration of the new page in the website gallery (via Actions' artifact or locally)
- [x] Make sure the email to covidrcontest@erum.io was received
- [x] Add a field with the current `date` and approx. merge time (e.g. `2020-04-01-16:30` for April 1st, 4:30 PM)
- [x] Squash-merge the pull request (commit headline will be OK, consider cleaning-up the details)
- [x] The initial comment should be automatically created, otherwise comment manually with the raw content of [vote-comment.md](../tree/master/contest/_tools/vote-comment.md) and an initial like
- [x] Provide here the link to the contribution as published to the gallery: https://milano-r.github.io/erum2020-covidr-contest/antoine-soetewey-how-to-create-a-simple-covid-19-dashboard.html
- [x] Instruct contributors on how to include their badge from the gallery page, share on social media, and get a [free ticket](https://www.eventbrite.it/e/covidr-erum-2020-pre-conference-event-tickets-104133026686).
- [x] Consolidate e-mailed information.

### CODE REVIEW

**Comment by riccardoporreca** [comment]
@AntoineSoetewey, thank you for your submission.

Your "Top R Resources" article was a great inspiration for the CovidR contest :clap:, and quite a few of those resources have been actually submitted to the contest.

The article itself seems however "just" an R markdown document listing contributions, where actually R is not really used for the article itself, therefore we cannot consider it suitable for the contest.

Still, we are aware of other nice work you have done and posted in your blog, which we believe would be very suitable / relevant as submission to CovidR, e.g.
- https://www.statsandr.com/blog/covid-19-in-belgium/
- https://www.statsandr.com/blog/how-to-create-a-simple-coronavirus-dashboard-specific-to-your-country-in-r/

We will be happy to include a contributions of yours in the contest!

**Comment by AntoineSoetewey** [comment]
I totally understand, it is indeed a collection and R is not used for the article.

I'll change the pull request with the article on [how to create a simple Covid-19 dashboard](https://www.statsandr.com/blog/how-to-create-a-simple-coronavirus-dashboard-specific-to-your-country-in-r/).

Feel free to edit it.
Thanks in advance!

File: contest/contributions/antoine-soetewey-how-to-create-a-simple-covid-19-dashboard.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Contribution's title, author, abstract
+title: How to create a simple Coronavirus dashboard specific to your country in R
```
Comment (riccardoporreca) [inline contest/contributions/antoine-soetewey-how-to-create-a-simple-covid-19-dashboard.yml]:
```suggestion
title: How to Create a Simple Coronavirus Dashboard Specific to your Country in R
```

File: contest/contributions/antoine-soetewey-how-to-create-a-simple-covid-19-dashboard.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+author: Antoine Soetewey
+abstract: |
+  This article will help you to build a visually appealing dashboard about the spread of COVID-19 Coronavirus specific to a country in R using flexdashboard.
+
+# Short name to be used in the website Gallery menu (can be the same as title)
+menu_entry: How to create a simple Coronavirus dashboard specific to your country in R
+# URL of the public source code repository
+repository: https://github.com/AntoineSoetewey/statsandr/blob/master/content/blog/2020-03-23-how-to-create-a-simple-coronavirus-dashboard-specific-to-your-country-in-r.html
```
Comment (riccardoporreca) [inline contest/contributions/antoine-soetewey-how-to-create-a-simple-covid-19-dashboard.yml]:
```suggestion
repository: https://github.com/AntoineSoetewey/statsandr/blob/master/content/blog/2020-03-23-how-to-create-a-simple-coronavirus-dashboard-specific-to-your-country-in-r.Rmd
```

File: contest/contributions/antoine-soetewey-how-to-create-a-simple-covid-19-dashboard.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Contribution's title, author, abstract
+title: How to create a simple Coronavirus dashboard specific to your country in R
+author: Antoine Soetewey
+abstract: |
+  This article will help you to build a visually appealing dashboard about the spread of COVID-19 Coronavirus specific to a country in R using flexdashboard.
+
+# Short name to be used in the website Gallery menu (can be the same as title)
+menu_entry: How to create a simple Coronavirus dashboard specific to your country in R
```
Comment (riccardoporreca) [inline contest/contributions/antoine-soetewey-how-to-create-a-simple-covid-19-dashboard.yml]: Shorter entry
```suggestion
menu_entry: "Country-specific Coronavirus Dashboard How-To"
```

**Review by riccardoporreca — APPROVED** [review]
Technical acceptance review

File: contest/contributions/antoine-soetewey-how-to-create-a-simple-covid-19-dashboard.yml [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+# URL of the public source code repository
+repository: https://github.com/AntoineSoetewey/statsandr/blob/master/content/blog/2020-03-23-how-to-create-a-simple-coronavirus-dashboard-specific-to-your-country-in-r.html
+# Type of contribution and how it can be featured as gallery content
+# Keep only the type and content relevant to your contribution
+# For a Shiny app, provide its URL:
+type: webpage
+content: https://www.statsandr.com/blog/how-to-create-a-simple-coronavirus-dashboard-specific-to-your-country-in-r/ # e.g. my-app on shinyapps.io
```
Comment (riccardoporreca) [inline contest/contributions/antoine-soetewey-how-to-create-a-simple-covid-19-dashboard.yml]: Include acceptance `date`
```suggestion
content: https://www.statsandr.com/blog/how-to-create-a-simple-coronavirus-dashboard-specific-to-your-country-in-r/
date: 2020-05-22-13:30
```

**Comment by riccardoporreca** [comment]
@AntoineSoetewey, your submission is now accepted and visible at https://milano-r.github.io/erum2020-covidr-contest/antoine-soetewey-how-to-create-a-simple-covid-19-dashboard.html :tada: 
You can click on the **eRum2020::Covidr badge** in the page to obtain the badge code you can embed in your repository.

Furthermore, feel free to spread your contribution on social media with the link above, using hashtags `#erum2020` `#covidr`, and invite the community to like the contribution using the corresponding button.

Last but not least: You can get your **free ticket for the pre-conference event** at https://www.eventbrite.it/e/covidr-erum-2020-pre-conference-event-tickets-104133026686

**Pull request merged**